types/nodejs-captcha/nodejs-captcha-tests.ts

```diff
@@ -0,0 +1,19 @@
+import captcha = require('nodejs-captcha');
+
+// test type exports
+type Opts = captcha.Options;
+type Cap = captcha.Captcha;
+
+const generatedCaptcha = captcha(); // $ExpectType Captcha
+captcha({ charset: ['1', '2', '3'] }); // $ExpectType Captcha
+captcha({ charset: ['1', '2', '3'] as const }); // $ExpectType Captcha
+captcha({ height: 300 }); // $ExpectType Captcha
+captcha({ width: 500 }); // $ExpectType Captcha
+captcha({ length: 30 }); // $ExpectType Captcha
```

Comment:
This shouldn't throw an error? Even if the test doesn't catch it might be inconsistent with the type definitions, just an option btw